core/src/main/java/org/hibernate/ogm/dialect/impl/OgmDialect.java

```diff
@@ -31,6 +32,7 @@
 
 	public OgmDialect(GridDialect gridDialect) {
 		this.gridDialect = gridDialect;
+		registerColumnType( Types.BINARY,"binary" );
```

Comment:
missing space between BINARY, and "binary".
Maybe also an in line comment like: "enabling support for binary types"